applications/openshift/scc/scc_limit_net_raw_capability/rule.yml

```diff
@@ -27,7 +27,26 @@ ocil_clause: 'NET_RAW does not exist or is configured in defaultAddCapabilities
 ocil: |-
     Inspect each SCC returned from running the following command:
     <pre>$ oc get scc</pre>
-    Review each SCC and determine that <tt>NET_RAW</tt> is either
+    Review each SCC and determine that either <tt>NET_RAW</tt> or <tt>ALL</tt> is either
     completely disabled as a list entry under <tt>requiredDropCapabilities</tt>,
-    or that <tt>NET_RAW</tt> is only enabled to a small set of
+    or that either <tt>NET_RAW</tt> or <tt>ALL</tt> is only enabled to a small set of
     containers and SCCs.
+
+{{%- if product == "ocp4" %}}
+warnings:
+    - general: |-
+        {{{ openshift_cluster_setting("/apis/security.openshift.io/v1/securitycontextconstraints") | indent(8) }}}
+{{%- endif %}}
+
+template:
+    name: yamlfile_value
+    vars:
+        ocp_data: "true"
+        filepath: /apis/security.openshift.io/v1/securitycontextconstraints
+        yamlpath: ".items[:]['requiredDropCapabilities']"
+        check_existence: "at_least_one_exists"
+        entity_check: "at least one"
+        values:
+          - type: "string"
+            value: "NET_RAW"
+            value: "ALL"
```

Comment:
I'm unsure if the multiple values would work like this..but let's figure out the yamlpath first

Reply:
This should work
```
values:
   - value: "NET_RAW|ALL"
     operation: "pattern match"
```

Reply:
Hold on a second. What actually was meant here? Array should contain both or having any of them is OK?

Reply:
Either. The attribute describes capabilities that are dropped. At a minimum we want NEW_RAW to be dropped specifically, but if anyone drops ALL, that's fine, too.

Reply:
Then yes, this would work:
```
values:
   - value: "^NET_RAW|ALL$"
     operation: "pattern match"
```